EncodingPlugin.js

```diff
@@ -26,14 +32,20 @@ class EncodingPlugin {
     );
 
     compiler.hooks.compilation.tap(pluginName, (compilation) => {
-      ['jsonpScript', 'hotBootstrap'].forEach((id) => {
-        const hook = compilation.mainTemplate.hooks[id];
-        if (hook) {
-          hook.tap(pluginName, (s) =>
-            s.replace(/(["'])utf-8["']/gi, `$1${options.encoding}$1`)
-          );
-        }
-      });
+      if (isWebpack5) {
```

Comment:
not needed anymore https://github.com/webpack/webpack/commit/db36afea4dfe0b76e66c04cd9429194c3b140002